node/network/protocol/src/request_response/mod.rs

```diff
@@ -102,6 +102,19 @@ const STATEMENTS_TIMEOUT: Duration = Duration::from_secs(1);
 /// to have 3 slow nodes connected, to delay transfer for others by `STATEMENTS_TIMEOUT`.
 pub const MAX_PARALLEL_STATEMENT_REQUESTS: u32 = 3;
 
+/// Response size limit for responses of POV like data.
+///
+/// This is larger than `MAX_POV_SIZE` to account for protocol overhead and for additional data in
+/// `CollationFetching` or `AvailableDataFetching` for example. We try to err on larger limits here
+/// as a too large limit only allows an attacker to waste our bandwidth some more, a too low limit
+/// might have more severe effects.
+const POV_RESPONSE_SIZE: u64 = MAX_POV_SIZE as u64 + 10_000;
+
+/// Maximum response sizes for `StatementFetching`.
+///
+/// This is `MAX_CODE_SIZE` plus some additional space for protocol overhead.
+const STATEMENT_RESPONSE_SIZE: u64 = MAX_CODE_SIZE as u64 + 10_0000;
```

Comment:
`10_0000` is weirdly formatted, did you mean `10_000`?
```suggestion
const STATEMENT_RESPONSE_SIZE: u64 = MAX_CODE_SIZE as u64 + 10_000;
```

Reply:
oh shoot. I really did not think, I will need a thorough review on this one. Thanks for the catch!